packages/parser/src/pipeline/classic/dedup.ts

```diff
@@ -1,6 +1,6 @@
 import { Observable } from 'rxjs';
 
-const LINE_PARSER = /^(\d+)\/(\d+)\s+(\d+):(\d+):(\d+)\.(\d+)(?=\s)/;
+const LINE_PARSER = /^(.*)? {2}([A-Z_]+),(.+)\s*$/;
```

Comment:
With the new log timestamps:
`7/28/2024 14:36:55.287-4  SPELL_AURA_REMOVED`
We can actually just call
`new Date("7/28/2024 14:36:55.287-4")` and get a correct object even with TZ info

This leads to a lot of simplifications down the line; eventually we should not have any deps on moment or moment-timezone once i'm confident this is all working

Reply:
nice!